platform-cloud/docs/monitoring/run-details.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Run details"
+description: "Monitoring a Nextflow pipeline executed through Seqera Platform."
+date: "21 Apr 2023"
+tags: [logging, monitoring, execution]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Select a workflow run from the **Runs** list to open a run details page. 
+
+![Run details and progress overview](./_images/progress-bar.png)
+
+The top of the page contains basic run details and a progress overview for an at-a-glance view of the run's status:
+- View and copy the run ID, pipeline name and repository, pipeline work directory, compute environment, and launch date. 
+- Select the star icon to favorite the run and find it more easily via a filter view in the runs list later. 
+- Use the options menu to apply labels, relaunch, resume, or delete the run, or to save the run as a new pipeline. 
+
+Select the tabs below the workflow run progress bar to view further run details:
+- **Tasks**: View the status and progress of pipeline tasks and [processes](#processes), including extensive [task details](#tasks)
+- **Logs**: View and download the pipeline run's execution log
+- **Metrics**: View resource [metrics](#metrics) for the run 
+- 
+- [Run information](#run-information) with real-time Nextflow execution details
+- [General summary](#general-summary) and [task status](#task-status)
+- List of pipeline 
+- [Aggregated stats](#aggregate-stats), [load](#load), and [utilization](#utilization)
+
+<Tabs>
+<TabItem value="Tasks" label="Tasks" default>
+
+![Task status overview](./_images/task-status-tiles.png)
+
+The cards at the top of the **Tasks** tab provide a real-time status of all tasks in the pipeline run:
```

Comment:
I propose removing the screenshot above and having a screenshot of just the icons from the Tasks tab over here.